specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimnotifications.json

```diff
@@ -402,11 +405,11 @@
         "tags": [
           "NotificationRecipientEmail"
         ],
-        "operationId": "NotificationRecipientEmail_Get",
+        "operationId": "NotificationRecipientEmail_CheckEntityExists",
```

Comment:
>NotificationRecipientEmail_CheckEntityExists [](start = 24, length = 44)

dont change the operationId... since this is stable api. we will change this in the next version 2018-01-01